CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to the Kowainik repositories
+
+## :wave: Greetings Traveler!
+
+We are delighted you're reading this, and we appreciate the effort you're
+taking to make our projects awesome! :sparkles:
+
+## How to contribute
+
+### :bug: Report bugs or feature request :bulb:
+
+If you discover a bug or have any proposals on how to make this project better
+don't hesitate to create an issue [here](../../issues/new) in a free format.
```

Comment:
I'm afraid, such links won't work anymore. Should we omit them?

Reply:
Let's keep them for now and see whether they work. If not -- we will rephrase :slightly_smiling_face: